dist/javascript.js

```diff
@@ -91,6 +91,14 @@ return JSON.stringify({
         };
 
         return noscript_data;
-    })()
+    })(),
 
+    bundler: (() => {
+        const bundler = {
+            webpack: !(typeof webpackJsonp === "undefined" && typeof webpackChunk === "undefined"),
+            parcel: !(typeof parcelRequire === "undefined")
```

Comment:
Just to confirm, when detected these are all expected to be in the global scope?

Also I only see `webpackJsonp` in the Wappalyzer implementation.

Reply:
Either or. So for Webpack, either `webpackJsonp` or `webpackChunk`. I created a [PR](https://github.com/wappalyzer/wappalyzer/pull/6309) for Wappalyzer but it is still awaiting review. 

It is also not a 100% solution because I have come across implementations where the global variable has a different name, ex: `webpackChunkMyProject`.

Reply:
Thanks for clarifying. I suppose it's possible to get all properties on the window object and test if any are prefixed with `webpackChunk` but that may be overkill. This is ok to merge.